yarn.lock

```diff
@@ -4805,6 +4805,49 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@metamask-previews/assets-controllers@npm:47.0.0-preview-33c13522":
+  version: 47.0.0-preview-33c13522
+  resolution: "@metamask-previews/assets-controllers@npm:47.0.0-preview-33c13522"
+  dependencies:
+    "@ethereumjs/util": "npm:^8.1.0"
+    "@ethersproject/abi": "npm:^5.7.0"
+    "@ethersproject/address": "npm:^5.7.0"
+    "@ethersproject/bignumber": "npm:^5.7.0"
+    "@ethersproject/contracts": "npm:^5.7.0"
+    "@ethersproject/providers": "npm:^5.7.0"
+    "@metamask/abi-utils": "npm:^2.0.3"
+    "@metamask/base-controller": "npm:^7.1.1"
+    "@metamask/contract-metadata": "npm:^2.4.0"
+    "@metamask/controller-utils": "npm:@metamask-previews/controller-utils@11.4.5-preview-33c13522"
```

Comment:
HERE - the controller utils has unreleased changes that we need in the assets preview package.